src/instructlab/model/evaluate.py

```diff
@@ -529,6 +522,8 @@ def evaluate(
         merge_system_user_message,
     )
 
+    validate_model(model)
```

Comment:
Could this go before we try to initialize the evaluator above?

Reply:
My logic was get_evaluator checks whether the model param was set.  So it made sense to check whether it was valid after that step.

Let me try again to move the validate into get_evaluator.  That's probably the best solution other than splitting get_evaluator up into a two different functions.

Reply:
Updated to move everything into get_evaluator.  As I did the work I remembered the reason the other reason I put it was because it was a lot more code to put it into get_evaluator and to rework the tests.  But it is a better change now I think.  I also added validate for judge models and base models which was missing before.